README.md

```diff
@@ -221,25 +221,26 @@ Create a task manifest file (e.g. `elasticsearch-task.json`).
     "workflow": {
         "collect": {
             "metrics": {
-                "/intel/elasticsearch/cluster/unassigned_shards": {},
-                "/intel/elasticsearch/cluster/active_shards": {},
-                "/intel/elasticsearch/node/*/indices/docs/count": {},
-                "/intel/elasticsearch/node/*/indices/merges/current_size_in_bytes": {},
-                "/intel/elasticsearch/node/*/indices/search/open_contexts": {},
-                "/intel/elasticsearch/node/*/jvm/mem/heap_used_in_bytes": {},
-                "/intel/elasticsearch/node/*/os/load_average": {},
-                "/intel/elasticsearch/node/*/thread_pool/fetch_shard_started/completed": {},
-                "/intel/elasticsearch/node/*/thread_pool/force_merge/threads": {},
-                "/intel/elasticsearch/node/*/transport/tx_size_in_bytes": {}
+                "/intel/elasticsearch/node/*/timestamp": {},
+		        "/intel/elasticsearch/node/*/host": {},
+                "/intel/elasticsearch/node/*/indices/merges/total_throttled_time_in_millis": {},
+                "/intel/elasticsearch/node/*/jvm/classes/total_loaded_count": {},
+                "/intel/elasticsearch/node/*/os/mem/total_in_bytes": {},
+                "/intel/elasticsearch/cluster/status": {}
+            },
+            "config": {
+                "/intel/elasticsearch/node": {
+                    "server": "192.168.99.100",
+                    "port": 9200
+                }
             },
-            "config": {},
             "process": [
                 {
                     "plugin_name": "passthru",
                     "process": null,
                     "publish": [
                         {                         
-                            "plugin_name": "file",
+                            "plugin_name": "mock-file",
```

Comment:
In general we shouldn't point users to mock plugins.  Mocks are used to test the framework.